public/js/actions/subscriptions.js

```diff
@@ -98,11 +100,20 @@ export function _createSubscription({dispatch, productId,
     console.log('Successfully subscribed + completed payment');
     dispatch(transactionActions.complete({userEmail: email}));
   }).fail($xhr => {
-    if (data.pay_method_nonce) {
-      dispatch({
-        type: actionTypes.CREDIT_CARD_SUBMISSION_ERRORS,
-        apiErrorResult: $xhr.responseJSON,
-      });
+    console.log($xhr);
```

Comment:
Is this still needed? I've actually been wondering if we should log requests more verbosely (in the `fetch()` wrapper) because it's really annoying when you hit a bug and then realize you forgot to turn on XHR logging in the devtools.